lib/active_model/serializer/adapter/json.rb

```diff
@@ -47,6 +26,151 @@ def fragment_cache(cached_hash, non_cached_hash)
           Json::FragmentCache.new().fragment_cache(cached_hash, non_cached_hash)
         end
 
+        private
+
+        def serialize_association(association)
+          serialized_or_virtual_of(association.serializer, association.options)
+        end
+
+        # iterate through the associations on the serializer,
+        # adding them to the parent as needed (as singular or plural)
+        #
+        # restrict_to is a list of symbols that governs what
+        # associations on the passed in seralizer to include
+        def add_resource_relationships(parent, serializer, restrict_to = [])
```

Comment:
We have the `Association` and it architecture on AMS, so I'd go for association if this it related to it, otherwise we can consider `relationship`